cron/duo_logpull.py

```diff
@@ -69,18 +72,21 @@ def process_events(mozmsg, duo_events, etype, state):
     # After processing these fields, we just pour everything into the "details" fields of Mozdef, except for the
     # noconsume fields.
 
-    if etype == 'administration':
-        noconsume = ['timestamp', 'host', 'action']
-    elif etype == 'telephony':
-        noconsume = ['timestamp', 'host', 'context']
-    elif etype == 'authentication':
-        noconsume = ['timestamp', 'host', 'eventtype']
+    if etype == "administration":
+        noconsume = ["timestamp", "host", "action"]
+    elif etype == "telephony":
+        noconsume = ["timestamp", "host", "context"]
+    elif etype == "authentication":
+        noconsume = ["timestamp", "host", "eventtype"]
     else:
         return
 
     # Care for API v2
-    if isinstance(duo_events, dict) and 'authlogs' in duo_events.keys():
-        duo_events = duo_events['authlogs']
+    if isinstance(duo_events, dict) and "authlogs" in duo_events.keys():
+        offset = duo_events["metadata"]["next_offset"]
```

Comment:
this is the new offset logic so that we save the offset sent by duo when we're done processing this batch of logs, only authentication supports this right now